src/main/scala/org/themaniacs/core/proxy/ClientProxy.scala

```diff
@@ -0,0 +1,11 @@
+package org.themaniacs.core.proxy
+
+import net.minecraftforge.fml.common.event.{FMLInitializationEvent, FMLPreInitializationEvent}
+
+class ClientProxy extends Proxy{
+  override def init(event: FMLInitializationEvent): Unit ={
```

Comment:
I'd personally rely on type inference for the return value instead of specifying it here.